python/python/glide/async_commands/core.py

```diff
@@ -5380,6 +5380,86 @@ async def getex(
             await self._execute_command(RequestType.GetEx, args),
         )
 
+    async def sscan(
+        self,
+        key: str,
+        cursor: str,
+        match: Optional[str] = None,
+        count: Optional[int] = None,
+    ) -> List[Union[str, List[str]]]:
+        """
+        Iterates incrementally over a set.
+
+        See https://valkey.io/commands/sscan for more details.
+
+        Args:
+            key (str): The key of the set.
+            cursor (str): The cursor that points to the next iteration of results. A value of "0" indicates the start of
+                the search.
+            match (Optional[str]): The match filter is applied to the result of the command and will only include
+                strings that match the pattern specified. If the set is large enough for scan commands to return only a
+                subset of the set then there could be a case where the result is empty although there are items that
+                match the pattern specified. This is due to the default `COUNT` being `10` which indicates that it will
+                only fetch and match `10` items from the list.
+            count (Optional[int]): `COUNT` is a just a hint for the command for how many elements to fetch from the set.
```

Comment:
Why COUNT in caps?

Reply:
That's how it is in Java so I'm just staying consistent.

Reply:
In java we have options, so user have another view on the docs...
Maybe you can tell
> Backed by Redis API `COUNT` keyword, which is just a hint ...

Or something likes. Anyway, it is up to you. There is no time to polish these docs now, unfortunately.